lib/broadside/target.rb

```diff
@@ -93,5 +93,16 @@ def to_h
         Cluster: @cluster
       }
     end
+
+    def update_safe_service_config
+      attributes_disallowed_for_updates = %i(
+        role
+        load_balancers
+      )
+
+      service_config.delete_if do |k, _|
```

Comment:
also more importantly, with activesupport i think you can just do `service_config.except(attributes_disallowed_for_updates)`